modules/tutorials/pages/migration-tutorial/sql-migration-tutorial-capella.adoc

```diff
@@ -0,0 +1,236 @@
+= Migrating your Data from MySQL to Capella
+:description: Using MySQL as a starting point, this guide demonstrates \
+how to migrate your existing data from SQL tables to a Couchbase Capella instance.
+:page-topic-type: guide
+:page-pagination: full
+
+[abstract]
+{description}
+
+== Introduction
+
+In this tutorial, 
+you will make use of  `cbimport` to migrate an example test database from MySQL to Couchbase Capella.
+
+== Prerequisites
+
+Before you make a start, you will need a Capella instance to run the tutorial.
+If you do not currently have an instance, 
+then you can create a Capella trial here =>{nbsp}https://www.couchbase.com/downloads/?family=capella[Capella Trial]
+
+You will also need to download and install the Server Development Tools package, 
+which includes the `cbimport` command line application.
+You will find installation instructions for the Server Development Tools package here =>{nbsp}xref:cli:cli-intro.adoc#server-developer-tools-package[Server Development Tools package]
+
+If you're running through the examples, 
+then you will also need an existing MySQL installation with the preexisting table structure 
+defined in xref:student-record-sql-database-section[the following section].
+
```

Comment:
Is it worth mentioning that the examples assume a Unix-type OS, and if you're unfortunate enough to be running Windows you should adjust the file paths accordingly?

I don't have a strong opinion either way, but we've definitely had people ask for this in other docs - however, given WSL, and some ppl running Powershell and not WSL, actually providing an alternative is a bit of a non-starter. :sweat_smile: